### ISSUE TITLE
Ссылка на Пикабу сдохла

### ISSUE THREAD

**Issue by Verity-Freedom** [anticensority/runet-censorship-bypass#311]
![image](https://github.com/user-attachments/assets/7bd747a2-3c15-432e-8fad-8ccf77beefe5)
Вот эта ссылочка умерла.

В принципе доступна по Вебархиву. Но с моей точки зрения там и так довольно устаревшая информация конечно.

**Comment by ilyaigpetrov** [comment]
Спасибо.
Если вы хотите охватить русскоязычную аудиторию, то писать на английском необязательно.
Также не всем очевидно, что нужно перейти в "Releases / Выпуски" для скачивания.

**Comment by Verity-Freedom** [comment]
> Также не всем очевидно, что нужно перейти в "Releases / Выпуски" для скачивания.

Согласен, добавил первой строчкой в readme. Не у всех они даже и есть, на самом деле. Например в Zapret у bol-van их нет.
